_docs/floodgate/setup.md

```diff
@@ -33,15 +35,15 @@ For BungeeCord/Velocity setups: you only are required to install Floodgate on th
 
 ### Installing Floodgate also on Spigot servers behind BungeeCord or Velocity
 
-This is only needed when you want to use the Floodgate API on your Spigot server(s) behind a proxy.
+This is only needed when you want to use the Floodgate API on your backend server(s) behind a proxy.
 
 - Install Floodgate on the proxy and on *all* backend servers as per the [previous instructions](/floodgate/setup/)
-- Enable `ip_forward` in your BungeeCord `config.yml` if using BungeeCord
-- Set `bungeecord` to `true` in your `spigot.yml`
+- Enable `ip_forward` in your BungeeCord `config.yml` if using BungeeCord. For Velocity, set up "player-info-forwarding".
+- Set `bungeecord` to `true` in your `spigot.yml`. When using Velocity, see their [guide](https://docs.papermc.io/velocity/player-information-forwarding), as they have multiple options that require different setups.
```

Comment:
What modes does floodgate work with in terms of velocity? Apart from modern

Reply:
hm, honestly? no clue.. but considering that forwarding works with older server versions, i'd assume that legacy would work too. Atleast from their wiki page it seems like legacy just uses the bungee way of forwarding, not fully sure though. BungeeGuard works on bungee networks, i dont see a reason why that should be different on Velocity - 
Sorry, but i really dont quite know :/